templates/miq-template.yaml

```diff
@@ -367,7 +436,81 @@ objects:
                 cpu: "${POSTGRESQL_CPU_REQ}"
               limits:
                 memory: "${POSTGRESQL_MEM_LIMIT}"
-
+- apiVersion: v1
+  kind: "DeploymentConfig"
+  metadata:
+    name: "${HTTPD_SERVICE_NAME}"
+    annotations:
+      description: "Defines how to deploy ManageIQ httpd"
+  spec:
+    strategy:
+      type: "Recreate"
+    triggers:
+      -
+        type: "ImageChange"
+        imageChangeParams:
+          automatic: true
+          containerNames:
+            - "httpd"
+          from:
+            kind: "ImageStreamTag"
+            name: "httpd:${HTTPD_IMG_TAG}"
+      -
+        type: "ConfigChange"
+    replicas: 1
+    selector:
+      name: "${HTTPD_SERVICE_NAME}"
+    template:
+      metadata:
+        name: "${HTTPD_SERVICE_NAME}"
+        labels:
+          name: "${HTTPD_SERVICE_NAME}"
+      spec:
+        serviceAccount: miq-sysadmin
+        serviceAccountName: miq-sysadmin
+        volumes:
+          -
+            name: "miq-httpd-volume"
+            persistentVolumeClaim:
+              claimName: "${NAME}-${HTTPD_SERVICE_NAME}"
+          -
+            name: "${NAME}-assets"
+            persistentVolumeClaim:
+              claimName: "${NAME}-assets"
+        containers:
+          -
+            name: "httpd"
+            image: "${HTTPD_IMG_NAME}:${HTTPD_IMG_TAG}"
+            ports:
+              - containerPort: 80
+              - containerPort: 443
+            readinessProbe:
+              timeoutSeconds: 1
+              initialDelaySeconds: 5
+              tcpSocket:
+                port: 443
+            livenessProbe:
+              timeoutSeconds: 1
+              initialDelaySeconds: 30
+              tcpSocket:
+                port: 443
+            volumeMounts:
+              -
+                name: "miq-httpd-volume"
+                mountPath: "/persistent"
+              -
+                name: "${NAME}-assets"
+                mountPath: "/persistent-assets"
+            env:
+              -
+                name: "APPLICATION_DOMAIN"
```

Comment:
upcoming ext-auth scripts, essentially the externally seen FQDN of the ManageIQ image.